python/pyspark/sql/context.py

```diff
@@ -287,6 +293,62 @@ def applySchema(self, rdd, schema):
         df = self._ssql_ctx.applySchemaToPythonRDD(jrdd.rdd(), schema.json())
         return DataFrame(df, self)
 
+    def createDataFrame(self, rdd, schema=None, samplingRatio=None):
```

Comment:
@rxin  Is there a better name for this? `createDataFrame` is still too long (longer than 'applySchema')